pkg/koyeb/utils.go

```diff
@@ -69,3 +72,29 @@ func loadYaml(file string) (string, error) {
 	}
 	return "", errors.New("Unknown format")
 }
+
+func watchDeployment(h *ServiceHandler, deploymentId string) {
```

Comment:
I think it would be nice to return an error if:
- we are unable to be healthy after timeout
- we are not able to fetch the deployment

And catching this and logging a message in the caller it needed.
Something like 'timeout waiting for healthy status', then continue with display.

Reply:
Maybe also return an error if the deployment is in error.

The function name would be more like:
`waitDeploymentHealthy(x, id) error`